src/cryojax/simulator/_pose.py

```diff
@@ -161,13 +160,6 @@ class EulerAnglePose(AbstractPose, strict=True):
         The sequence of axes over which to apply
         rotation. This is a string of 3 characters
         of x, y, and z. By default, `'zyz'`.
```

Comment:
Perhaps still include some details about the convention you use (at high level natural language explanation). They can look at your code to see what it does in more detail...